iot-hub/Samples/device/ConventionBasedOperations/Thermostat/ThermostatSample.cs

```diff
@@ -277,5 +286,24 @@ private static double GenerateTemperatureWithinRange(int max = 50, int min = 0)
         {
             return Math.Round(s_random.NextDouble() * (max - min) + min, 1);
         }
+
+        private async Task UpdateForEmptyWritableProperty(CancellationToken cancellationToken)
+        {
+            const string propertyName = "targetTemperature";
+
+            var reportedProperties = new ClientPropertyCollection();
+
+            // If the writable properties are empty, report the default value with ACK(ac=203, av=0) as part of the PnP convention.
+            // "DefaultPropertyValue" is set from the device when the desired property is not set via the hub.
+            var propertyValue = _deviceClient.PayloadConvention.PayloadSerializer.CreateWritablePropertyResponse(
+                DefaultPropertyValue, 203, 0);

```

Comment:
can we add the 203 to `CommonClientResponseCodes` ?

Reply:
We may leave "203" as in the samples for now, and change it to something like `CommonClientResponseCodes.Default` after we update status codes in the class `CommonClientResponseCodes`. Are we considering to add other codes [here](https://github.com/Azure/azure-iot-sdk-csharp/blob/preview/shared/src/CommonClientResponseCodes.cs)?